CONTRIBUTING.rst

```diff
@@ -9,13 +9,9 @@ Code style
 ----------
 
 The documentation style is described at
-https://github.com/matrix-org/matrix-doc/blob/master/meta/documentation_style.rst.
+https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst.
 
-Python code within the ``matrix-doc`` project should follow the same style as
-synapse, which is documented at
-https://github.com/matrix-org/synapse/tree/master/docs/code_style.md.
```

Comment:
this was a bit of a dead link, and we don't really have any python code here any more.